loadtest/store/memstore/store.go

```diff
@@ -238,7 +238,7 @@ func (s *MemStore) UserForPost(postId string) (string, error) {
 	if post, ok := s.posts[postId]; ok {
 		return post.UserId, nil
 	}
-	return "", nil
+	return "", ErrPostNotFound
```

Comment:
Should we also make the same change in `UserForPost` and `FileInfoForPost` as well ?

Reply:
We can certainly do that :+1: